### PR TITLE
[DR-1412] Add scheduled message in my plan

### PR DESCRIPTION
**Pull request by mvillaseco**
# Overview
 Show to the user a scheduled warning message

# Changes:
 - Add message
 - Mobile ready

### CODE REVIEW

File: src/wwwroot/controllers/PlanCtrl.js [hunk 1]
```diff
@@ -47,6 +47,11 @@
         vm.allowChangePlan = !response.data.endDate;
         vm.hideDragMe = !vm.isFreeTrial;
         defaultPlanDeliveries = !vm.isFreeTrial ? response.data.includedDeliveries.toString() : defaultPlanDeliveries; 
+        if (!vm.allowChangePlan) {
```
Comment (andresmoschini) [inline src/wwwroot/controllers/PlanCtrl.js]: I would rename it as _hasScheduledPlan_

File: src/wwwroot/controllers/PlanCtrl.js [hunk 2]
```diff
@@ -47,6 +47,11 @@
...
+          settings.getNextPlan().then(function(response){
+            vm.nextPlan = response.data.includedDeliveries;
```
Comment (andresmoschini) [inline src/wwwroot/controllers/PlanCtrl.js]: It is not the name of the plan, what happen if it is PRO or something like that?

File: src/wwwroot/services/settings.js [hunk 3]
```diff
@@ -181,6 +182,13 @@
         url: url
       });
     }
+    function getNextPlan() {
+      return $http({
+        actionDescription: 'action_getting_current_plan',
+        method: 'GET',
+        url: RELAY_CONFIG.baseUrl + '/accounts/' + auth.getAccountName() + '/agreements' + '/next'
```
Comment (andresmoschini) [inline src/wwwroot/services/settings.js]: I think that ` + '/agreements/next'` could be better if we search in the code-

File: src/wwwroot/controllers/PlanCtrl.js (hunk 2 above)
Comment (mvillaseco) [reply src/wwwroot/controllers/PlanCtrl.js]: Yes, we are going to show only the amount of deliveries allowed for the plan the user select, we are currently not showing the name of the current plan.

Comment (andresmoschini) [reply src/wwwroot/controllers/PlanCtrl.js]: In that case maybe _nextPlan_ is not the best name

**Pull request merged**